### PR TITLE
Update aad0e77b-e5db-474b-b041-0a43ee04baf4.md

### PR DESCRIPTION
**Pull request by michbeck**
Fixed AdyenExecute3DStep.php code example

### CODE REVIEW

File: 202001.0/aad0e77b-e5db-474b-b041-0a43ee04baf4.md [hunk 1]
```diff
@@ -55,13 +55,7 @@ class AdyenExecute3DStep extends AbstractBaseStep
...
     public function requireInput(AbstractTransfer $quoteTransfer)
     {
-        if ($quoteTransfer->getPayment()->getPaymentSelection() === AdyenConfig::ADYEN_CREDIT_CARD &&
-            $this->config->isAdyenCreditCard3dSecureEnabled()
-        ) {
-            return true;
-        }
- 
-        return false;
+        return ($this->isAdyenCreditCardPayment($quoteTransfer) && $this->config->isAdyenCreditCard3dSecureEnabled() && $quoteTransfer->getPayment()->getAdyenRedirect());
```
Comment (kkicheglovspryker) [inline 202001.0/aad0e77b-e5db-474b-b041-0a43ee04baf4.md:58]: convert to multi-line

File: 202001.0/aad0e77b-e5db-474b-b041-0a43ee04baf4.md [hunk 2]
```diff
@@ -86,6 +80,16 @@ class AdyenExecute3DStep extends AbstractBaseStep
             'fields' => $quoteTransfer->getPayment()->getAdyenRedirect()->getFields(),
         ];
     }
+    
+    /**
+     * @param \Spryker\Shared\Kernel\Transfer\AbstractTransfer $quoteTransfer
+     *
+     * @return bool
+     */
+    private function isAdyenCreditCardPayment(AbstractTransfer $quoteTransfer): bool
```
Comment (kkicheglovspryker) [inline 202001.0/aad0e77b-e5db-474b-b041-0a43ee04baf4.md:89]: protected

File: 202001.0/aad0e77b-e5db-474b-b041-0a43ee04baf4.md (hunk 1 above)
Comment (michbeck) [reply 202001.0/aad0e77b-e5db-474b-b041-0a43ee04baf4.md]: Updated

File: 202001.0/aad0e77b-e5db-474b-b041-0a43ee04baf4.md (hunk 2 above)
Comment (michbeck) [reply 202001.0/aad0e77b-e5db-474b-b041-0a43ee04baf4.md]: Updated

**Review by kkicheglovspryker — APPROVED** [review]

**Review by AsonUnique — APPROVED** [review]

**Review by abitskil — APPROVED** [review]

**Pull request merged**